plugins/org.eclipse.fordiac.ide.subapptypeeditor/src/org/eclipse/fordiac/ide/subapptypeeditor/editors/SubAppNetworkBreadCrumbEditor.java

```diff
@@ -206,14 +208,18 @@ public boolean outlineSelectionChanged(Object selectedElement) {
 			if (selectedElement instanceof final FBNetwork fbn) {
 				selectedElement = fbn.eContainer();
 			}
-			if ((selectedElement instanceof FBNetworkElement) || (selectedElement instanceof SubAppType)) {
+			if ((selectedElement instanceof FBNetworkElement) || (selectedElement instanceof SubAppType)
+					|| (selectedElement instanceof VarDeclaration)) {
 				EObject refElement = null;
 				if (selectedElement instanceof final FB fb) {
 					refElement = fb;
 					selectedElement = refElement.eContainer().eContainer();
 				} else if (selectedElement instanceof final SubApp subApp) {
 					refElement = subApp;
 					selectedElement = refElement.eContainer().eContainer();
+				} else if (selectedElement instanceof final VarDeclaration varDecl) {
+					refElement = varDecl;
+					selectedElement = varDecl.eContainer().eContainer();
```

Comment:
how do you know that the vardeclaration is not in the interface of the subapp type?

Reply:
Ah thx for the clarfication I missed that line.